### PR TITLE
issue1 - fixing blueprint template so it runs

### PR DESCRIPTION
**Pull request by bfitzpat**
-- upgrading to camel 2.18.2
-- fixing template pom and camel config
-- updated versions and indent issues after code review

Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

### CODE REVIEW

**Comment by bfitzpat** [comment]
Now this thing actually runs. Huzzah! Build a new blueprint project, do "mvn install" and "mvn camel:run" and you should see the same.

File: app/index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var path = require('path');
 var mkdirp = require('mkdirp');
 const utils = require('./util');
 
-const defaultCamelVersion = "2.18.1";
+const defaultCamelVersion = "2.18.2";
```
Comment (apupier) [inline app/index.js]: why are we using this "old" Camel version? What about using latest one (2.22.1)? or latest corresponding to a Fuse version(2.21.0)?

File: app/templates/blueprint/pom.xml [hunk 2]
```diff
@@ -187,15 +201,41 @@
 					<encoding>UTF-8</encoding>
 				</configuration>
 			</plugin>
+			
+			<!-- to generate the MANIFEST.MF of the bundle -->
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<version>${version.maven-bundle-plugin}</version>
+      <extensions>false</extensions>
```
Comment (apupier) [inline app/templates/blueprint/pom.xml]: indentation issue

File: app/templates/blueprint/pom.xml [hunk 3]
```diff
@@ -187,15 +201,41 @@
...
+            </goals>
+          </execution>
+        </executions>
+ 			</plugin>
 
+    <!-- to include MANIFEST.MF in the bundle -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>2.6</version>
```
Comment (apupier) [inline app/templates/blueprint/pom.xml]: latest version is 3.1.0 https://maven.apache.org/plugins/maven-jar-plugin/

**Comment by bfitzpat** [comment]
@apupier - the generated project doesn't build with the 3.1 version of the maven-jar-plugin 
`Plugin org.apache.maven.plugins:maven-jar-plugin:3.1 or one of its dependencies could not be resolved: Could not find artifact org.apache.maven.plugins:maven-jar-plugin:jar:3.1`

This worked with the older version (2.6) - should we go back to that?

**Comment by bfitzpat** [comment]
Just tried that locally. Version 3.0.2 seems to exist in public repos more readily than 3.1.0 and builds locally. And it still runs locally with camel version 2.22.1. I'll drop back to 3.0.2 for now unless you have any concerns.

**Pull request merged**